src/aioswitcher/bridge.py

```diff
@@ -173,13 +173,15 @@ class SwitcherBridge:
     def __init__(
         self,
         on_device: Callable[[SwitcherBase], Any],
-        broadcast_ports: list[int] = [SWITCHER_UDP_PORT_TYPE1, SWITCHER_UDP_PORT_TYPE2],
+        broadcast_ports: Union[List[int], None] = None,
     ) -> None:
         """Initialize the switcher bridge."""
+        if broadcast_ports is None:
+            broadcast_ports = [SWITCHER_UDP_PORT_TYPE1, SWITCHER_UDP_PORT_TYPE2]
```

Comment:
Why not assigning them in line 176?

Reply:
https://github.com/TomerFi/aioswitcher/pull/559#discussion_r979384122

Reply:
This is not related, you can still assign them in Python 3.9

Reply:
it doesn't really matters...
assigning in row 180 or 176, as I explained above this was reported as a codesmell by sonarlint.
this might be true or might be a false-positive,
I could either investigate why is this a false-positive, change this code to adhere the warning, or disable this specific sonarlint rule.
I chose to make the change and not disable the rule because it makes absolutely not difference from the code perspective.
but... if you're worried by this small meaningless modification, I can reverse it and disable sonarlint...
your call.